README.md

```diff
@@ -22,6 +22,35 @@ cls.fit(X_train, y_train)
 predictions = cls.predict(X_test)
 ```
 
+## auto-sklearn acceleration with sklearnex
+
+You can accelerate auto-sklearn with [Intel(R) Extension for Scikit-Learn (sklearnex)](https://github.com/intel/scikit-learn-intelex). The acceleration is achieved through **patching**: replacing scikit-learn algorithms with their optimized versions provided by the extension.
+
+Read [system requirements](https://intel.github.io/scikit-learn-intelex/system-requirements.html) and install sklearnex with pip or conda:
+```bash
+pip install scikit-learn-intelex
+conda install scikit-learn-intelex
+conda install -c conda-forge scikit-learn-intelex
```

Comment:
do we need to have conda-forge line here?

Reply:
conda-forge is mostly popular as main channel, so mention about sklearnex availability at conda-forge is useful for many users who don't like to complicate environment with packages from 2 and more channels

Reply:
It's ok to mention, but i don't think it worth adding command example as it's add complexity - let's pick single line that we will give users - for now we are putting lot of details in to readme. 

@outoftardis  - can we insert copy button in readme?   i.e. instead of select - right click - copy click  we will use just copy button

Reply:
@napetrov it's already there :) in the right upper corner

![image](https://user-images.githubusercontent.com/17865025/143430989-2aeca8d9-9180-4e4a-a666-8a6a872a14c6.png)

Reply:
then we for sure need single line not multiple lines in block

Reply:
Lines are splitted now

Reply:
oh, forgot that it is about .md, I leaved one line at README, .rst format at web site lacks copy button, but I think separate lines look better